geoSegment/geoSegment.js

```diff
@@ -167,26 +169,56 @@ async function emptyOrMakeOutputDirectories(segments) {
  * @param {ScheduledSessionData} scheduledSessionData
  * @param {SessionSeriesData} sessionSeries
```

Comment:
Could you add `segments` to the function signature here

Reply:
yes